hw3/style-hw3.css

```diff
@@ -0,0 +1,457 @@
+	height: 370px;
+	background: url(image/bcg-banner.png);
+	background-size: cover;
+	position: relative;
+}
+
+.banner__images {
+	display: inline-block;
+	width: 50%;
+	min-width: 650px;
+	height: 100%;
+	position: relative;
+}
+
+.banner__img {
+	position: absolute;
+}
+
+.banner__img-1 {
+	left: 13%;
+	top: 31%;
+	z-index: 4;
+	animation-name: appearance-third;
+	animation-duration: 4s;
+	animation-timing-function: ease-in;
+}
+
+.banner__img-2 {
+	left: 9%;
+	top: 19%;
+	z-index:3;
+	animation: appearance-second;
+	animation-duration: 4s;
+	animation-timing-function: ease-in;
+}
+
+.banner__img-3 {
+	left: 25%;
+	top: 9%;
+	z-index:2;
+	animation-name: appearance-first;
+	animation-duration: 4s;
+	animation-timing-function: ease-in;
+}
+
+.banner__info {
+	padding: 75px 0 25px;
+	margin-left: auto;
+	width: 30%; 
+	height: 370px;
+	display: inline-block;
+	vertical-align: top;	
+}
+
+.banner-container {
+	height: 280px;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+}
+
+.banner__title {
+	text-transform: capitalize;
+	color: #fff;
+	font: bold 26px "Droid sans", sans-serif;
+	text-shadow: -1px 1px 2px #000;
+}
+
+.banner__text {
+	color: #fff;
+	font: 16px/26px Arial, sans-serif;
+}
+
+.banner__button {
+	width: 190px;
+	height: 55px;
+	font: 28px/17px "Droid sans", sans-serif;
+	margin: 0 auto;
+}
+
+@keyframes appearance-first {
+    from {
+	opacity: 0;
+    }
+   	33% {
+  opacity: 1;
+   	}
+} 
+
+@keyframes appearance-second {
+    from {
+	opacity: 0;
+    }
+    33% {
+  opacity: 0;
+    }
+    66% {
+  opacity: 1;
+    }	
+}
+
+@keyframes appearance-third {
+    from {
+	opacity: 0;
+    }
+  66% {
+  opacity: 0;
+    }	
+    to {
+	opacity: 1;
+    }	
+}
+
+/*------subscription------*/
+.subscription {
+	width: 80%;
+	margin: 70px auto 0 auto;
+	padding: 0 10px;
+	background: linear-gradient(to top, #d2d2d2, #e8e7e5);
+	border-radius: 4px;
+	min-height: 70px;
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	align-items: center;
+	position: relative;
+}
+ 
+.subscription:before,
+.subscription:after {
+  content: "";
+  position: absolute;
+  z-index: -2;
+  bottom: 15px;
+  left: 20px;
+  width: 25%;
+  height: 20%;
+  max-width: 300px;
+  box-shadow: 0 15px 20px rgba(0, 0, 0, 0.4);
+  transform: rotate(-3deg);
+}
+ 
+.subscription:after {
+  right: 20px;
+  left: auto;
+  transform: rotate(3deg);
+}
+
+.subscription__text {
+	color: #000;
+	font: 16px/26px Arial, sans-serif;
+}
+
+.subscription__email {
+	height: 42px;
+	border: none;
+	border-radius: 4px;
+	padding-left: 15px;
+}
+
+.subscription__submit {
+	width: 62px;
+	height: 42px;
+	font: 14px/26px "Droid sans", sans-serif;
+}
+
+/*--------preview---------*/ 
+.preview__center {
+	width: 80%;
+	margin: 60px auto 0 auto;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+}
+
+.preview__logo {
+ vertical-align: middle;
+}
+
+.preview__item {
+	width: 24%;
+	min-width: 285px;
+	margin-right: 70px;
```

Comment:
why do you need them if you are using flexboxes? remove them, better have min-width for flex-container